cli/tests/test_global_opts.rs

```diff
@@ -53,6 +53,8 @@ fn test_version() {
     let expected = [
         "jj ?.??.?\n",
         "jj ?.??.?-????????????????????????????????????????\n",
+        // Allow users of Nix to build with a dirty (uncommitted) revision
+        "jj ?.??.?-?irty\n",
```

Comment:
nit: perhaps the commit description isn't updated? It currently says "allow ... for debug builds".